src/components/tool/guide/Config/ConfigMultipleNumbers.js

```diff
@@ -0,0 +1,98 @@
+import { CheckCircleIcon, WarningIcon } from "@chakra-ui/icons";
+import {
+  Alert,
+  AlertIcon,
+  Box,
+  CloseButton,
+  Flex,
+  Heading,
+  Square,
+  Tooltip,
+} from "@chakra-ui/react";
+
+import { useState } from "react";
+
+import InputNumberStepper from "../../../input/InputNumberStepper";
+
+export default function ConfigMultipleNumbers({
+  title,
+  description,
+  selected,
+  value = {},
+  onChange,
+  inputProps,
+  showAlert = false,
+  alertDismissible = true,
+  alertType = "info",
+  alertMsg = "",
+}) {
+  const [alertDismissed, setAlertDismissed] = useState(false);
+  const onClose = () => {
+    setAlertDismissed(true);
+  };
+  const keyToReadable = (key) => {
+    return key
+      .replace(/([A-Z])/g, " $1") // Add a space before each uppercase letter
+      .replace(/^./, (char) => char.toUpperCase()); // Capitalize the first letter
+  };
+  return (
+    <Box>
+      <Flex direction="row" width="100%" align="center">
+        <Box w="120px">
+          {selected ? (
+            <CheckCircleIcon color="green.500" />
+          ) : (
+            <Tooltip label="Pick an option">
+              <WarningIcon color="orange.500" />
+            </Tooltip>
+          )}
+          <Tooltip label={description} placement="top" hasArrow>
+            <Heading size="xs" mb="0" display="inline" ml="2">
+              {title}
+            </Heading>
+          </Tooltip>
+        </Box>
+        <Square size="10px" />
+        <Flex>
+          {Object.keys(value).map((key, index) => (
+            <InputNumberStepper
+              key={key}
+              w="100px"
+              label={keyToReadable(key)}
+              value={value[key]}
+              onChange={(newValue) => {
+                onChange({
+                  ...value,
+                  [key]: newValue,
```

Comment:
Updated values get stored as strings, which messes with the validation in the `show` method.

This is also true for ConfigDuration, but that doesn't break validation because of coercion during comparison in JS :D

We can fix that by replacing ConfigDuration with the new component 

```suggestion
                  [key]: parseInt(newValue),
```

Reply:
Fixed